java.java

```diff
@@ -0,0 +1,11 @@
+// package descrition
+import java.io.*;
+
+/** 
+ * docummentation for java class
+ * multiline docummentation goes ere
+ */
+class HelloWorldApp {
+    /* multiline coment goes here
```

Comment:
Typo inside comment in word `coment`. Possible corrections: cement (60), comment (44), moment (43),